muta_apm/src/lib.rs

```diff
@@ -0,0 +1,39 @@
+//!
+
+use std::borrow::Cow;
+
+use rustracing::sampler::AllSampler;
+use rustracing_jaeger::reporter::JaegerCompactReporter;
+pub use rustracing_jaeger::span::StartSpanOptions;
+use rustracing_jaeger::Tracer;
+
+const SPAN_CHANNEL_SIZE: usize = 20;
+const _REPORTER_BUFFER_SIZE: usize = 5;
+
+pub struct MutaTracer {
+    pub inner: Tracer,
+}
+
+impl MutaTracer {
+    pub fn new() -> Self {
+        let (span_tx, span_rx) = crossbeam_channel::bounded(SPAN_CHANNEL_SIZE);
+
+        std::thread::spawn(move || {
+            let reporter = JaegerCompactReporter::new("muta-chian").unwrap();
```

Comment:
https://docs.rs/rustracing_jaeger/0.4.1/rustracing_jaeger/reporter/struct.JaegerCompactReporter.html#errors

Consider the case where tracing is not enabled